### ISSUE TITLE
Permit multiple owners of a note

### ISSUE THREAD

**Issue by rastapopougros** [hackmdio/codimd#1174]
It will be very usefull to let a note have multiple owners. So we all could change the read and write permissions, or any other config for one particular note.

Context : we are a small team with our own codimd instance, so it's not really the same as in #12, no need to have multiple groups in the same instance (but if it's implemented, we could have one only group in our instance, why not). When someone in the team is not here, and it's him/her who created a note, the others must be able to change the note config, that's the important need.

In the config panel of a note, a selector should let us add other owners.

**Comment by jackycute** [comment]
Maybe you're looking for #35?

**Comment by rastapopougros** [comment]
Mmh not the same need at all, it seems :) 

This issue don't focus on edit the note, but being able to edit the config of the note = being the owner or the admin (have the admin rights).

Maybe describe a persona use case : 
- Hi, I'm RastaPopoulos and I just created a new note, I can change it's permissions and maybe other configs later
- @jackycute can write in the note
- Tomorrow, I take some vacations, or my computer will crash, or any reason making me unavailable
- I want @jackycute and maybe other colleagues being full admin of this note *as much as me*
- Also, when a future version will add user page and list notes by owners, the same note must be listed for all users who are owners (we all must see this note in  a list "My notes")

**Comment by SISheogorath** [comment]
@rastapopougros what kind of note config details are you talking about? iirc the only things an owner can do, that everyone else can't do is changing the restrictions for view and deleting a note.

**Comment by rastapopougros** [comment]
> changing the permissions and deleting a note

Like it's a trivial thing! :p
It's an huge responsability, and this is exactly what I'm talking about since this begining. Also, maybe in future versions there will be more configs, but only those for restrictions are sufficient to really need the sharing of those confs between multiple owners.

Also there is the link between a note and who is the owner(s)/admin(s). In future version, if we finally have a page which answers the question "what are the notes I manage, where I'm admin ?", so the "multiple admins feature" will permit to list the *same* note, in this list of multiple users. (And having the list for where we are owner/admin is really critical as long as the number of notes grows and grows.)

**Comment by SISheogorath** [comment]
Agreed! Was just wondering in first place "how big the issue is". I totally agree with you, I just think we have to get to the level were groups exist as a concept in CodiMD first.

**Comment by rastapopougros** [comment]
Yeah, kind of agree. Like I said from the first post, if there is a group concept, we can deal having one only group for our team in our instance, and it will do the job. :)

**Comment by Wurzelmann** [comment]
Is there any news to this? It would be really useful, especially when working in teams.
